resources/qml/voip/CallInviteBar.qml

```diff
@@ -0,0 +1,95 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.3
+import QtQuick.Dialogs 1.3
+import QtQuick.Layouts 1.2
+import im.nheko 1.0
+import "../"
+
+Rectangle {
+
+    visible: CallManager.haveCallInvite
+    color: "#2ECC71"
+    implicitHeight: visible ? rowLayout.height + 8 : 0
+
+    MessageDialog {
+        id: warningDialog
+        icon: StandardIcon.Warning
+    }
+
+    RowLayout {
+        id: rowLayout
+
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.verticalCenter: parent.verticalCenter
+        anchors.leftMargin: 8
+
+        Avatar {
+            width: avatarSize
+            height: avatarSize
+            url: CallManager.callPartyAvatarUrl.replace("mxc://", "image://MxcImage/")
+            displayName: CallManager.callParty
+        }
+
+        Label {
+            font.pointSize: fontMetrics.font.pointSize * 1.1
+            text: "  " + CallManager.callParty + " "
```

Comment:
Are you padding this on both sides by adding spaces? Or are those there for a different reason? In the former case, you can use the following usually: https://doc.qt.io/qt-5/qml-qtquick-layouts-layout.html#leftMargin-attached-prop

Reply:
A vestige from when this existed in C++. Will fix.